content/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ssl/README.md

```diff
@@ -0,0 +1,806 @@
+
+Install cert-manager using `helm` as follows:
+
+```bash
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm upgrade cert-manager jetstack/cert-manager \
+    --install \
+    --create-namespace \
+    --wait \
+    --namespace cert-manager \
+    --set installCRDs=true
+```
+
+This will create three Deployments and some Services and Pods in a new namespace called `cert-manager`.
+It also installs various cluster scoped supporting resources such as RBAC roles and Custom Resource Definitions.
+
+You can view some of the resources that have been installed as follows:
+
+```bash
+kubectl -n cert-manager get all
+```
+
+And you can explore the Custom Resource Definitions (cert-manager's API) using `kubectl explain`, as follows:
+
+```bash
+kubectl explain Certificate
+kubectl explain CertificateRequest
+kubectl explain Issuer
+```
+
+> 📖 Read about [other ways to install cert-manager](../../installation).
+>
+> 📖 Read more about [Certificates and Issuers](../../concepts).
+
+## Create a test ClusterIssuer and a Certificate
+
+Now everything is ready for you to create your first certificate.
+This will be a self-signed certificate but later we'll replace it with a Let's Encrypt signed certificate.
+
+```yaml
+# clusterissuer-selfsigned.yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: selfsigned
+spec:
+  selfSigned: {}
+```
+🔗 <a href="clusterissuer-selfsigned.yaml">`clusterissuer-selfsigned.yaml`</a>
+
+```bash
+kubectl apply -f clusterissuer-selfsigned.yaml
+```
+
+Then use `envsubst` to substitute your chosen domain name into the following Certificate template:
+
+```yaml
+# certificate.yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: www
+spec:
+  secretName: www-tls
+  privateKey:
+    rotationPolicy: Always
+  commonName: www.$DOMAIN_NAME
+  dnsNames:
+    - www.$DOMAIN_NAME
+  usages:
+    - digital signature
+    - key encipherment
+    - server auth
+  issuerRef:
+    name: selfsigned
+    kind: ClusterIssuer
+```
+🔗 <a href="certificate.yaml">`certificate.yaml`</a>
+
+```bash
+envsubst < certificate.yaml | kubectl apply -f -
+```
+
+Use `cmctl status certificate` to check the status of the Certificate:
+
+```bash
+cmctl status certificate www
+```
+
+If successful, the private key and the signed certificate will be stored in a Secret called `www-tls`.
+You can use `cmctl inspect secret www-tls` to decode the base64 encoded X.509 content of the Secret:
+
+```terminal
+$ cmctl inspect secret www-tls
+...
+Valid for:
+        DNS Names:
+                - www.cert-manager-tutorial-22.site
+        URIs: <none>
+        IP Addresses: <none>
+        Email Addresses: <none>
+        Usages:
+                - digital signature
+                - key encipherment
+                - server auth
+...
+```
+
+> 🔗 If you don't already have `envsubst` installed you can [download and install a Go implementation of `envsubst`](https://github.com/a8m/envsubst).
```

Comment:
suggestion: Would this tip be more helpful if it were moved up closer to where we actually use envsubst for the first time?